apps/emails-and-messages/src/modules/mjml/get-active-mjml-settings.ts

```diff
@@ -0,0 +1,43 @@
+import { AuthData } from "@saleor/app-sdk/APL";
+import { appRouter } from "../trpc/trpc-app-router";
+import { logger as pinoLogger } from "../../lib/logger";
+
+interface GetMjmlSettingsArgs {
+  authData: AuthData;
+  channel: string;
+  configurationId: string;
+}
+
+export const getActiveMjmlSettings = async ({
+  authData,
+  channel,
+  configurationId,
+}: GetMjmlSettingsArgs) => {
+  const logger = pinoLogger.child({
+    fn: "getMjmlSettings",
+    channel,
+  });
+
+  const caller = appRouter.createCaller({
+    appId: authData.appId,
+    saleorApiUrl: authData.saleorApiUrl,
+    token: authData.token,
+    ssr: true,
+  });
```

Comment:
This should not work. `protectedClientProcedure` should check for client token. You provide backend token here